### ISSUE TITLE
Sign-in form is not accessible

### ISSUE THREAD

**Issue by lindsaykwardell** [elk-zone/elk#53]
Two big things I noticed on the sign in form:

- [x] It's not a form, so pressing Return/Enter doesn't cause the login to initiate.
- [x] It doesn't capture focus within the dialog, or follow some other dialog guidelines.

I'm happy to take this one on, just reporting it here.

https://user-images.githubusercontent.com/32229300/203848518-a628170a-195c-4925-b668-86aa5bd7281a.mov
